src/App/App.tsx

```diff
@@ -35,6 +36,14 @@ const NonprofitInfo = lazy(() => import("pages/informational/NonprofitInfo"));
 const DonorInfo = lazy(() => import("pages/informational/DonorInfo"));
 
 export default function App() {
+  /**
+   * ping AWS api every 5 minutes,
+   * this invokes token refresh and fires refresh events: tokenRefresh | tokenRefresh_failure.
+   * if tokenRefresh_failure, `A.` user state is set to null
+   * when `A.` happens and user is on a protected page (wrapped with `withAuth` HOC),
```

Comment:
when user is not on a protected page, should not see any change - unless he then visits a protected page (will be redirected to signin)